tools/tentacle_creator/tentacle_creator.py

```diff
@@ -36,21 +36,13 @@ def __init__(self, config):
 
     @staticmethod
     def get_template_path(name):
-        return "{0}/{1}/{2}/{3}{4}{5}".format(TOOLS_PATH,
-                                              TENTACLE_CREATOR_PATH,
-                                              TENTACLE_TEMPLATE_PATH,
-                                              name,
-                                              TENTACLE_TEMPLATE_PRE_EXT,
-                                              TENTACLE_TEMPLATE_EXT)
+        return f"{TOOLS_PATH}/{TENTACLE_CREATOR_PATH}/{TENTACLE_TEMPLATE_PATH}/{name,}{TENTACLE_TEMPLATE_PRE_EXT}" \
+            f"{TENTACLE_TEMPLATE_EXT}"
 
     @staticmethod
     def get_config_template_path(name):
-        return "{0}/{1}/{2}/{3}{4}{5}".format(TOOLS_PATH,
-                                              TENTACLE_CREATOR_PATH,
-                                              TENTACLE_TEMPLATE_PATH,
-                                              name,
-                                              TENTACLE_CONFIG_TEMPLATE_PRE_EXT,
-                                              TENTACLE_TEMPLATE_EXT)
+        return f"{TOOLS_PATH}/{TENTACLE_CREATOR_PATH}/{TENTACLE_TEMPLATE_PATH}/{name}" \
```

Comment:
I think that a [_Multi-line string syntax_](https://www.smallsurething.com/multi-line-strings-in-python/) will be better here. (with `"""` )